src/content/en/fundamentals/performance/critical-rendering-path/measure-crp.md

```diff
@@ -5,20 +5,20 @@ description: Learn to measure the critical rendering path.
 {# wf_updated_on: 2014-09-17 #}
 {# wf_published_on: 2014-03-31 #}
 
-# Measuring the Critical Rendering Path with Navigation Timing {: .page-title }
+# Measuring the Critical Rendering Path {: .page-title }
 
 {% include "web/_shared/contributors/ilyagrigorik.html" %}
 
-You can't optimize what you can't measure. Thankfully, the Navigation 
-Timing API gives us all the necessary tools to measure each step of the 
-critical rendering path!
+You can't optimize what you can't measure. To measure your Critical Rendering
+Path, instrument your code with the Navigation Timing API, or audit your app
+with Lighthouse.
 
-### TL;DR {: .hide-from-toc }
-- Navigation Timing provides high resolution timestamps for measuring CRP.
-- Browser emits series of consumable events which capture various stages of the CRP.
+## Instrumenting your code with the Navigation Timing API {: #navigation-timing }
 
-
-The foundation of every solid performance strategy is good measurement and instrumentation. Turns out, that is exactly what the Navigation Timing API provides.
+The foundation of every solid performance strategy is good measurement and
+instrumentation. The Navigation Timing API provides high-resolution timestamps
+for measuring CRP, and the browser emits a series of consumable events which
+capture various stages of the CRP.
```

Comment:
Hmm, what do you think of rewording this a bit to highlight that NT is all about RUM tracking? Something like..

"The combination of Navigation Timing API, which provides a mechanism to gather high-resolution data for a number of important page load milestones, and other events emitted by the browser as the page is loading, enables us to gather and track the real-user performance of the critical rendering path."